docs/services/jupyterlab.md

```diff
@@ -65,6 +65,27 @@ Then with that virtual environment activated, you can run the command to create
     If the uenv is not present in the local repository, it will be automatically fetched.
     As a result, JupyterLab may take slightly longer than usual to start.
 
+
+## Using Julia in JupyterHub
+
+Each time you start a JupyterHub server, you need to do the following in the JupyterHub Spawner Options form mentioned above:
+!!! important "pass a [`julia`][ref-uenv-julia] uenv and the view `jupyter`."
+
+At first time use of Julia within Jupyter, IJulia and one or more Julia kernel needs to be installed. Type the following command in a shell within JupyterHub to install IJulia, the default Julia kernel and, on systems whith Nvidia GPUs, a Julia kernel running under Nvidia Nsight Systems:
```

Comment:
Please add line breaks between sentences, as per the contributing guide:
https://eth-cscs.github.io/cscs-docs/contributing/#text-formatting

Reply:
done